src/main/java/uk/gov/justice/probation/courtcaseservice/controller/CourtCaseController.java

```diff
@@ -307,7 +309,15 @@ public ResponseEntity<CaseListResponse> getCaseList(
         final var createdBeforeOrDefault = Optional.ofNullable(createdBefore)
                 .orElse(LocalDateTime.of(MAX_YEAR_SUPPORTED_BY_DB, 12, 31, 23, 59));
 
-        var courtCases = courtCaseService.filterHearings(courtCode, date, createdAfterOrDefault, createdBeforeOrDefault);
+        final var caseSearchFilter = CaseSearchFilter.builder()
+                .courtCode(courtCode)
+                .createdAfter(createdAfterOrDefault)
+                .createdBefore(createdBeforeOrDefault)
+                .source(source)
+                .build();
+
+        var courtCases = courtCaseService.filterHearings(caseSearchFilter);
```

Comment:
I think it is better to name the filter 'HearingSearchFilter' as we are not filtering the case but the hearings. no?